src/util/psbt/error.rs

```diff
@@ -98,6 +101,7 @@ impl fmt::Display for Error {
             }
             Error::NoMorePairs => f.write_str("no more key-value pairs for this psbt map"),
             Error::HashParseError(e) => write!(f, "Hash Parse Error: {}", e),
+            Error::MissingUtxo => f.write_str("Utxo is missing"),
```

Comment:
Rewording suggestion: "UTXO information is not present in PSBT"
This conveys a bit more context that may be needed for debugging.

Reply:
Sorted